src/wallet/api/wallet.cpp

```diff
@@ -942,6 +942,12 @@ uint64_t WalletImpl::approximateBlockChainHeight() const
 {
     return m_wallet->get_approximate_blockchain_height();
 }
+
+uint64_t WalletImpl::estimateBlockChainHeight() const
+{
+	return m_wallet->estimate_blockchain_height();
```

Comment:
It's nice to try and keep the same style as surrounding code. Note how the function above and below have a different indent width. Sorry to be pedantic :)